app/Http/Controllers/Member/ListMemberController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Http\Controllers\Members;
+
+use App\Models\Member;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+
+class ListMemberController extends Controller
+{
+    public function __invoke(string $squadUuid)
+    {
+        $member = Member::query()->where('squad_uuid', $squadUuid)->get();
+
+        return response()->json($member);
```

Comment:
Tratamento de resultado vazio pra em casos que nenhum membro for encontrado, retornar uma mensagem explicando a ausência de membros pode melhorar a UX

Reply:
Funcionalidade não  incluida na master do hyperF, podemos escrever na develop